src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -245,7 +245,10 @@ class ActivityFeed extends React.Component<Props, State> {
             ? errorMessageByEntryType[activeFeedEntryType]
             : undefined;
 
-        const isInlineFeedItemErrorVisible = !isLoading && activeFeedEntryType && !activeEntry;
+        // Annotation deeplink allows for activeFeedEntryType but no activeFeedEntryId
```

Comment:
Can we avoid this and keep the fallback path as `/activity`? Having one-off logic for annotations seems a little unfortunate.

Reply:
Although unfortunate, it feels like the more consistent pattern to allow the `sb` param to be the source of truth for `ContentPreview` and `ContentSidebar`. If we don't allow partial paths for annotations we run into issue when viewing previous versions annotations and representing the state when the selected annotation is deselected

Reply:
I see. Rather than using something annotations-specific, can we add a check for the existence of `activeFeedEntryId` as well?

```
const isInlineFeedItemErrorVisible = activeFeedEntryType && activeFeedEntryId && !activeEntry && !isLoading;